lib/Echidna/Config.hs

```diff
@@ -40,6 +46,11 @@ data EConfig = EConfig { _cConf :: CampaignConf
                        }
 makeLenses ''EConfig
 
+data EConfig2 = EConfig2 { _econfig :: EConfig
```

Comment:
Can we have a better name for `Econfig2`, maybe `EconfigInternal` or something like that?